aws/aws.go

```diff
@@ -232,10 +232,10 @@ func changeMessageRetentionPeriod(queueURL *string, newDuration time.Duration, s
 	return err
 }
 
-// changeVisibilityTimeout, Extends the message visibility for the whole queue
+// ensureVisibilityTimeout extends the message visibility for the whole queue
 // newDuration: The new value for the message's visibility timeout.
 // Value needs to be 0 to 43200 seconds.
-func changeVisibilityTimeout(queueURL *string, newDuration time.Duration, sqsClient sqsiface.SQSAPI) error {
+func ensureVisibilityTimeout(queueURL *string, newDuration time.Duration, sqsClient sqsiface.SQSAPI) error {
```

Comment:
You could probably remove these functions now that they've been combined in the single call. Are they being used elsewhere?

Reply:
Not at the moment. I figured in the future maybe it could be used somewhere. If we want to expose a client to change the subscriber timeout or retention specifically then we can.